### PR TITLE
Feature Add project files

### PR DESCRIPTION
**Pull request by joshuaoivie**
Dear Code Reviewer,

This pull request contains the following changes

-Add Readme
-Add Index.html
-Add Style.css

Thank you for your anticipated constructive review.

Warm Regards,
Joshu

### CODE REVIEW

**Review by joshuaoivie — COMMENTED** [review]
Dear Joshua,

Great work on your project so far! 🎉🎉 Here are some highlights from your project:
- You have used GitHub flow correctly
- There are no linter errors
- You have attributed the source of the project in your ReadMe


**Status: Required Changes ♻️⛔️**
You may need to make some changes to your project merging this pull request. Here are some changes to make:

- [ ] - Please ensure the link in your ReadMe to the live demo of your project is functional 

- [ ] - Please ensure that you use HTML tags correctly - see the inline comments below for more details.

You can reach out to me on Slack if you require any clarification.

Happy Coding 🥳

File: README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+## Built With
+
+- Html
+- Css
+- GitHub
+
+## Live Demo
+
+[Live Demo Link](https://joshuaivie.github.io/review-trial-project/)
```
Comment (joshuaoivie) [inline README.md:21]: Here is the link to the live version of your project. You may need to update to a functional link!

File: index.html [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+    <title>Form-basic</title>
+
+  </head>
+
+  <body>
+    <!-- Top level container -->
+    <div class="container">
+
+      <!-- PAGE HEADER -->
+      <DIV  class="header" >
```
Comment (joshuaoivie) [inline index.html:19]: Please endeavour to use HTML tags correctly. Tags in HTML 5 are written in lowercare

File: index.html [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+            name="age"
+            id="number"
+            min="10"
+            max="99"
+            class="form-control"
+            placeholder="Your age goes here "/>
+        </div>
+
+        <div class="form-group">
+          <SPAN >How do you get your income? </SPAN>
```
Comment (joshuaoivie) [inline index.html:66]: Here's another example of a tag that has been used incorrectly. Please update your project to use all HTML tags in lower case.

**Review by AleksandarIvezic — APPROVED** [review]
## Hi @joshuaivie 👋🏼 , 

You did a great job here. You showed that you learned how to use GitHub features for reviews and how to communicate in a professional way! 👌🏼 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 

gif

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**